twake/frontend/src/app/views/client/main-view/MainView.tsx

```diff
@@ -23,6 +24,8 @@ const MainView: FC<PropsType> = ({ className }) => {
   const workspaceId = useRouterWorkspace();
   const channelId = useRouterChannel();
 
+  useUserList();
```

Comment:
Could we add a comment here to specify why we do this ?

Reply:
Done